### ISSUE TITLE
New XP scaling -- impossible to level in some cases

### ISSUE THREAD

**Issue by BlizzardWizards** [Entroper/FF1Randomizer#41]
![ff1_0722d7a5_r---awbboo-1](https://user-images.githubusercontent.com/33459001/32501874-d3721fae-c39e-11e7-9e36-cade949ef917.png)

Seed/Flags: 0722D7A5 / r%%%Awbboo

I generated 5 seeds using these flags, all have the same problem. Upon hitting level 2 or 3, the Experience needed to level up is set to near max. I guess there is an underflow error when gaining too much XP relative to current level, possibly if you skip 1 or more levels worth of XP from a single fight? The screenshot attached is after fighting the first 5 Imp fight outside Coneria, gaining the level, then checking the Status screen.

Thank you!

**Comment by Entroper** [comment]
If you fight another battle, do you gain a level?

The original game had this bug if you gained enough exp for more than one level in a single battle.  It would only level you once, and then you'd be overflowed to get to the next level, but as soon as you fought another battle, you would gain a level and it would fix itself.

**Comment by sbq92** [comment]
I ran into this before the change in the way xp works and I leveled up just fine after another battle.

**Comment by BlizzardWizards** [comment]
Thanks for the comments guys. I continued gaining a level per battle until level 5, then the display was fixed.

This was from version 1.5.2., though I also saw it on 1.4.9 when I tried that one.

I swear, though I have no proof, that when this happened the first time subsequent fights were not fixing the problem and I could see the 65k+ remaining reduce by the amount I was gaining, but I didn't screenshot at that time and it's entirely possible that I'm just wrong.

Thanks again. I love this randomizer.
